config/settings.json

```diff
@@ -8,13 +8,13 @@
           "spack-config": "2024.03.22"
         },
         "0.22": {
-          "spack": "9653dbe2e20814d0b97393dce80af73c63d97300",
+          "spack": "f32c2593340d854b69494afe32cbef0b764933fe",
           "spack-config": "2025.02.1"
```

Comment:
We don't want to bump the minor field in the version?

Reply:
No. That's the spack-config tag. We are updating the Spack commit hash.